bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -2500,4 +2511,98 @@ protected function processAvailableFieldDefinitions($fds, &$firstOne, &$commonFi
             }
         }
     }
+    
+    /**
+     * @Route("/export-json-config", methods={"GET"})
```

Comment:
same as for import, maybe 
export-csv-import-config-from-json

Reply:
Done, renamed as "export-csv-import-config-as-json"